.github/workflows/ci.yml

```diff
@@ -43,5 +43,9 @@ jobs:
           echo "::set-env name=BYRON_CONFIGS::$Env:GITHUB_WORKSPACE\byron_configs"
         continue-on-error: true
       - run: npm install
-      - run: npm test
+      - run: npm test unit
+        continue-on-error: true
```

Comment:
This means that even if unit tests fail, the CI run will still pass.

Reply:
Ah got it, I need to learn more about the actions syntax. I thought this would allow the test suite to complete, but makes sense now. For that separate steps